prepare/templates/qa/contextual.py

```diff
@@ -10,6 +10,15 @@
     overwrite=True,
 )
 
+add_to_catalog(
+    MultiReferenceTemplate(
+        input_format="Question: {question}\nContext: {context}",
+        references_field="answers",
+    ),
+    "templates.qa.contextual.simple_question_first",
```

Comment:
I changed the [qa tasks](https://github.com/IBM/unitxt/blob/main/prepare/tasks/qa/tasks.py) for unitxt, so the names of the qa templates should be adjusted accordingly.

Reply:
note the answer field is `answer` for those tasks (unlike plural `answers` in fm-eval), also the name is `contextualized.abstractive` or `contextualized.extractive`.

Reply:
@matanor @eladven is there even extractive task fort generative models? Asking it to copy the input?